testkit/src/lib.rs

```diff
@@ -897,27 +886,29 @@ impl TestKit {
 ///         AfterCommitService { counter: Arc::new(AtomicUsize::default()) }
 ///     }
 ///
+/// #    pub fn new_instance(&self) -> Box<dyn Service> {
```

Comment:
Is this method used anywhere?

Reply:
yep, it's used by the `ServiceFactory` macro: `service_constructor = "Self::new_instance",`